app/js/controllers/actions/event/event-add-ctrl.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+angular
+.module('app.controllers')
+.controller('EventAddCtrl', function($scope, $rootScope, $stateParams, $state,
+		$interval, Restangular, apiDescriptor, formElementProvider, dataTransformer) {
+	apiDescriptor.then(function(apiDescription) {
```

Comment:
Can you move this to be below the declaration of the declaration of the scope variables that aren't functions? It would read better to indicate that the handler occurs *after* the code below has executed.